public/app/features/dashboard/components/ShareModal/ShareLink.tsx

```diff
@@ -133,31 +177,44 @@ export class ShareLink extends PureComponent<Props, State> {
             {isDashboardSaved && (
               <div className="gf-form">
                 <a href={imageUrl} target="_blank" rel="noreferrer" aria-label={selectors.linkToRenderedImage}>
-                  <Icon name="camera" /> Direct link rendered image
+                  <Icon name="camera" />
+                  <Trans id="share-modal.link.rendered-image">Direct link rendered image</Trans>
                 </a>
               </div>
             )}
 
             {!isDashboardSaved && (
-              <Alert severity="info" title="Dashboard is not saved" bottomSpacing={0}>
-                To render a panel image, you must save the dashboard first.
+              <Alert
+                severity="info"
+                title={t({ id: 'share-modal.link.save-alert', message: `Dashboard is not saved` })}
+                bottomSpacing={0}
+              >
+                <Trans id="share-modal.link.save-dashboard">
+                  To render a panel image, you must save the dashboard first.
+                </Trans>
               </Alert>
             )}
           </>
         )}
 
         {panel && !config.rendererAvailable && (
-          <Alert severity="info" title="Image renderer plugin not installed" bottomSpacing={0}>
-            <>To render a panel image, you must install the </>
-            <a
-              href="https://grafana.com/grafana/plugins/grafana-image-renderer"
-              target="_blank"
-              rel="noopener noreferrer"
-              className="external-link"
-            >
-              Grafana image renderer plugin
-            </a>
-            . Please contact your Grafana administrator to install the plugin.
+          <Alert
+            severity="info"
+            title={t({ id: 'share-modal.link.render-alert', message: `Image renderer plugin not installed` })}
+            bottomSpacing={0}
+          >
+            <Trans id="share-modal.link.render-instructions">
+              To render a panel image, you must install the &nbsp;
```

Comment:
There are two spaces at the end of this sentence, remove the one before `&nbsp;`